docroot/themes/custom/yukonca_glider/templates/paragraphs/paragraph--chart--default.html.twig

```diff
@@ -0,0 +1,96 @@
+ * - content: All paragraph items. Use {{ content }} to print them all,
+ *   or print a subset such as {{ content.field_example }}. Use
+ *   {{ content|without('field_example') }} to temporarily suppress the printing
+ *   of a given child element.
+ * - attributes: HTML attributes for the containing element.
+ *   The attributes.class element may contain one or more of the following
+ *   classes:
+ *   - paragraphs: The current template type (also known as a "theming hook").
+ *   - paragraphs--type-[type]: The current paragraphs type. For example, if the paragraph is an
+ *     "Image" it would result in "paragraphs--type--image". Note that the machine
+ *     name will often be in a short form of the human readable label.
+ *   - paragraphs--view-mode--[view_mode]: The View Mode of the paragraph; for example, a
+ *     preview would result in: "paragraphs--view-mode--preview", and
+ *     default: "paragraphs--view-mode--default".
+ * - view_mode: View mode; for example, "preview" or "full".
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_paragraph()
+ *
+ * @ingroup themeable
+ */
+#}
+
+<div class="chart">
+  <h2>{{ paragraph.field_title.value|raw }}</h2>
+  <figure class="paragraph-items content">
+    <div id="chart-graph-{{ paragraph.id() }}" class="chart-graph"></div>
+    <div class="chart-description">
+      <div aria-hidden="true" class="highcharts-linked-description">
+        <p>
+          <strong class="chart-stat">{{ paragraph.field_highlighted_stat.value|raw }}</strong>
+        </p>
+                          
+        <div class="field field-name-field-chart-description field-type-text-with-summary field-label-hidden">
+          <div class="field-items">
+            <div class="field-item even"> {{ paragraph.field_description.value|raw }} </div>
+          </div>
+        </div>
+      </div>
+      <section class="yukon-department-sections basic-page-collapsable">
+        <div class="panel-group" id="accordion-{{ paragraph.id() }}" role="tablist">
+          <div class="panel panel-default">
+            <div class="panel-heading" role="tab">
+              <h4 class="panel-title">
+                <a class="collapsed" data-toggle="collapse" data-parent="#accordion-{{ paragraph.id() }}" href="#accordion-content-{{ paragraph.id() }}" aria-expanded="false" aria-controls="accordion-content-{{ paragraph.id() }}">
+                  {{ 'Source and notes on data'|t }} <span class="float-right text-blue-400 title-icon"><i class="fa fa-plus-square"></i></span>
+                </a>
+              </h4>
+            </div>
+            <div id="accordion-content-{{ paragraph.id() }}" class="panel-collapse collapse" role="tabpanel" aria-labelledby="Employment-insurance-EI-beneficiaries-in-the-Yukon-by-gender" style="">
```

Comment:
The IDs have been fixed, but the `aria-labelledby` value is still "Employment-insurance-EI-beneficiaries-in-the-Yukon-by-gender". I don't know what the `aria-labelledby` value should be, but I'm pretty sure the current value is wrong.

Reply:
We have cleaned up the html code that we copied from the D7 version. Please check again.